<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidFilter</class>
 <widget class="QDialog" name="MidFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口大小：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BtnTrad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioBtnTen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十字型</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnRec">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>方形</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnCycle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>圆形</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnAdpt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自适应</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
